cmd/podman/attach.go

```diff
@@ -17,6 +17,7 @@ var (
 		RunE: func(cmd *cobra.Command, args []string) error {
 			attachCommand.InputArgs = args
 			attachCommand.GlobalFlags = MainGlobalOpts
+			attachCommand.Remote = remoteclient
```

Comment:
Why isn't this just a global flag?

Reply:
because that is only good in main. we may need it to extend into adapter where the cli is passed.